kubetest2/kubetest2-gce/deployer/up.go

```diff
@@ -79,6 +88,8 @@ func (d *deployer) verifyFlags() error {
 		return err
 	}
 
+	d.kubectl = filepath.Join(d.RepoRoot, "cluster", "kubectl.sh")
```

Comment:
should we fallback to kubectl in PATH ?

Reply:
I don't know. I opted for this because almost every script I've seen that needs to use kubectl uses this script in the repo (or at least defaults to it without a manual override).